content/docs/1.0.x/reference/api/index.md

```diff
@@ -59,7 +61,11 @@ The Keptn API is documented in terms of a [Swagger API documentation](https://sw
         link="./assets/swagger-example.png"
         caption="Keptn Swagger API documentation - Example"
         width="700px">}}
-
+        
+## Uploading an API file to a Keptn project 
+* An API file can be uploaded to a project in either of the following ways :
+    * Upload the file directly to your git repo.
+    * Use the `resource-service` API for uploading, as shown in the example snippet from above.
```

Comment:
```suggestion
* Use the `resource-service` API for uploading
using the Swagger API as shown above.
This gives you sections for uploading resources
on the project, stage or service level.

  To mimic what the `keptn add-resource` command does,
  use the `POST API` call.
```
You might want to grab the screenshot that Andi provided and insert it here but I'm not going to try to do that in the github editor ;-)